### ISSUE TITLE
Docs: SAR lambda layer documentation uses incorrect version of Powertools

### ISSUE THREAD

**Issue by svozza** [awslabs/aws-lambda-powertools-python#5359]
### What were you searching in the docs?

When attempting to deploy the SAR lambda layer in one of my accounts, I copied the CFN from the [docs](https://docs.powertools.aws.dev/lambda/python/latest/#sar) and it was pointing to `v3.0.0`:

```yaml
AwsLambdaPowertoolsPythonLayer:
      Type: AWS::Serverless::Application
      Properties:
          Location:
              ApplicationId: arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer-v3-python312-x86_64
              SemanticVersion: 3.0.0 # change to latest semantic version available in SAR
```

However, this resulted in a CFN error that said that the version was not available. Upon checking, it appears that SAR only supports v2 of Powertools currently.

### Is this related to an existing documentation section?

https://docs.powertools.aws.dev/lambda/python/latest/#sar

### How can we improve?

Set the version in the code example to  a 2.x version.

### Acknowledgment

- [x] I understand the final update might be different from my proposed suggestion, or refused.

**Comment by dreamorosi** [comment]
Hi @svozza, thank you for opening this issue.

We'll get this fixed and published soon.

In parallel, we'll also continue working to get v3 published in SAR (#5221).

**Comment by svozza** [comment]
Thank you!
